lib/v2/curius/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/links/:name', require('./links'));
```

Comment:
The path here should be the same as `maintainer.js` and `path` in the docs

Reply:
They're now the same.